src/main/java/duke/taskmanagement/DukeException.java

```diff
@@ -0,0 +1,45 @@
+package duke.taskmanagement;
+
+
+public class DukeException {
+    protected String typeOfTask;
+
+    /**
+     * Create new duke exception to handle
+     * the exception when running Duke.
+     * @param str indicate the type of task when handling this exception
+     */
+    public DukeException(String str){
+        this.typeOfTask = str;
+    }
+
+    /**
+     * Return the String that shows the error
```

Comment:
Perhaps the first sentence in the header comment starts with Returns and an empty line between description and parameter section is more appropriate?